beaglev.md

```diff
@@ -0,0 +1,392 @@
+BeagleV
+=======
+
+BeagleV is a low-cost RISC-V 64-bit based platform. The beta revision of the
+board uses the Starfive JH7100 SoC.
+
+How to build
+============
+
+```
+$ SHELL=/bin/bash kas-container build ./meta-riscv/beaglev.yml
+```
+
+The `kas-container` script can be obtained from
+[kas repository](https://github.com/siemens/kas/blob/2.4/kas-container).
+
+Build results
+=============
+
+- core-image-minimal-beaglev-starlight-jh7100.wic.gz
+- core-image-minimal-beaglev-starlight-jh7100.wic.bmap
+- fw_payload.bin
+- fw_payload.elf
+- extlinux.conf
+- Image
+- starfive_vic7100_beagle_v.dtb
+- u-boot.bin
+
+Flashing
+========
+
+* Flash `core-image-minimal-beaglev-starlight-jh7100.wic.gz` onto uSD card:
+
+```
+$ sudo bmaptol copy --bmap core-image-minimal-beaglev-starlight-jh7100.wic.bmap core-image-minimal-beaglev-starlight-jh7100.wic.gz /dev/sdx
+```
+
+* Flash U-Boot:
+
+```
+$ perl -e 'print pack("l", (stat @ARGV[0])[7])' fw_payload.bin > fw_payload.bin.out
+$ cat fw_payload.bin >> fw_payload.bin.out
+```
+
+* Interrupt the boot process and send the `fw_payload.bin.out`. Details can be
+  found [here](https://github.com/tpetazzoni/buildroot/blob/beaglev/board/beaglev/readme.txt#L60)
+
+* Alter the content of the first (`boot`) partition:
+
+```
+
+```
+
+Testing
+=======
+
+* The root partition is sometimes not found on time - `rootdelay=3` parameter
+  was added to the cmdline
```

Comment:
I still have issues with this not being found, is the rootdelay already included? (I see `rootdelay=2`)